### PR TITLE
4.2 - Rails-First-Step

### PR DESCRIPTION
**Pull request by easmith2**
Next, I'd like to try not having separate files for each post.

And I should have asked about this checkpoint this afternoon: 
"Checkpoint: implemented dynamic views (variables, template partial rendering)"
I have no idea if I did this. I didn't use variables, but I don't think I could explain "template partial rendering", so I'm going to say I probably didn't do it.

### CODE REVIEW

**Comment by rpearce** [comment]
`<%= render 'some_partial' %>` is what that means, and you did it

File: firststep/app/controllers/home_controller.rb [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+  def post_2
+    @title = 'My Less Cool Post'
+  end
+
+  def post_3
+    @title = 'mmm...Caaandy'
+  end
+
+  def post_4
+    @title = 'Why Joss, WHY?!?'
```
Comment (rpearce) [inline firststep/app/controllers/home_controller.rb]: Whedon?

File: firststep/app/controllers/home_controller.rb [hunk 2]
```diff
@@ -0,0 +1,27 @@
...
+  def index
+    @title = 'Surf & Paddle Co.'
+  end
+
+  def about
+    @title = 'About S & P'
+  end
+
+  def post_1
+    @title = 'Purple Lightsaber FTW'
```
Comment (rpearce) [inline firststep/app/controllers/home_controller.rb]: Mace ftw

**Comment by rpearce** [comment]
prodigious

File: firststep/app/controllers/home_controller.rb (hunk 1 above)
Comment (easmith2) [inline firststep/app/controllers/home_controller.rb]: To accompany all the Whedonisms in that post

**Pull request merged**